src/content/developers/docs/consensus-mechanisms/pow/index.md

```diff
@@ -94,6 +94,10 @@ At a high level, proof-of-stake has the same end goal as proof-of-work: to help
 - [Majority attack](https://en.bitcoin.it/wiki/Majority_attack)
 - [On settlement finality](https://blog.ethereum.org/2016/05/09/on-settlement-finality/)
 
+### Videos {#videos}
+
+ - [What is Proof of Work?](https://youtu.be/XLcWy1uV8YM)
```

Comment:
```suggestion
 - [What is Proof of Work (PoW)｜Explained For Beginners](https://www.youtube.com/watch?v=3EUAcxhuoU4)
```
Thanks for the contribution @MrBrain295. I think a video section is great as it allows for a few different learning styles, but would like to see a video that is a bit higher quality. This Binance video is very concise, has good audio, and while being concise it is thorough.

Reply:
Yeah agreed. The whiteboard crypto video is okay but I feel like this does a better job explaining the necessary information given where we expect a user to be at this point in their Ethereum learning journey.